HIP Documentation/HIP Programming Guide.rst

```diff
@@ -0,0 +1,23 @@
+=========================
+HIP Programing Guide
+=========================
+
+What is this repository for?
+
+HIP allows developers to convert CUDA code to portable C++. The same source code can be compiled to run on NVIDIA or AMD GPUs. 
+
+Key features 
+==============
+
+HIP is very thin and has little or no performance impact over coding directly in CUDA or hcc “HC” mode.
+
+HIP allows coding in a single-source C++ programming language including features such as templates, C++11 lambdas, classes, namespaces, 
+and more.
+
+HIP allows developers to use the “best” development environment and tools on each target platform.
+
+The “hipify” tool automatically converts source from CUDA to HIP.
```

Comment:
We have no such a tool as `hipify`, we have `hipify-clang` and `hipify-perl` together called [hipify tools](https://github.com/ROCm-Developer-Tools/HIPIFY#tools-to-translate-cuda-source-code-into-portable-hip-c-automatically).